### PR TITLE
docs upgrade to 0.0.16

### PR DESCRIPTION
**Pull request by GeorgeNash**
- Fix broken links
- Save legacy 0.0.15 docs
- Upgrade docs to 0.0.16

### CODE REVIEW

File: docs/_data/previous_versions.yml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 - title: 0.0.15
-  url: /docs
+  url: /docs/0_0_15
```
Comment (CharlieCrisp) [inline docs/_data/previous_versions.yml]: Does this need to be updated to actually include 0.0.16 as well. I can't quite remember how it works...

Comment (GeorgeNash) [reply docs/_data/previous_versions.yml]: yes

File: docs/doc_templates/future.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+programming syntax.
+
+Our overarching goal is to increase the scale and speed at which Bayesian inference can be run, and 
+the size of the datasets that can be assimilated to the point where these limitations rarely, if ever, 
+affect modellers. We also strive to achieve this without increasing the complexity of model logic.
+
+We already have a wide variety of probabilistic programming operators, multiple inference algorithms, 
+fast performance on the CPU, early-stage documentation and examples, and basic tooling to support 
+some modelling paradigms. However, the project is at a pre-alpha stage. There is still much work to do 
+and we are only able offer early adopters limited ad-hoc support.
```
Comment (migwellian) [inline docs/doc_templates/future.md]:
```suggestion
and we are only able to offer early adopters limited ad-hoc support. 
```

File: docs/doc_templates/future.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
+---
+permalink: /docs/future
+title: The future of Keanu
+---
+
+The aim of this project is to develop a probabilistic programming environment that enables modellers to
```
Comment (migwellian) [inline docs/doc_templates/future.md]: We use American spellings, right? That would be a single `l` - i.e. modelers, modeling - throughout.

**Pull request merged**